Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs

```diff
@@ -102,12 +102,14 @@ public WxeFunctionStateManager (HttpSessionStateBase session)
       ArgumentUtility.CheckNotNull("session", session);
       _session = session;
 
-      _functionStates = (Dictionary<string, WxeFunctionStateMetaData>)_session[s_sessionKeyForFunctionStates];
```

Comment:
I assume these changes are due to the upgrade in coreforms?

Reply:
Yeah the nullability of the session indexer changed